src/config.ts

```diff
@@ -161,7 +161,7 @@ export default {
   ENABLE_GEOLOCATION_LOGGING: ENABLE_GEOLOCATION_LOGGING === "true",
   ENABLE_GRAPHQL_UPLOAD: ENABLE_GRAPHQL_UPLOAD === "true",
   ENABLE_GRAVITY_MARKETING_COLLECTIONS:
-    ENABLE_GRAVITY_MARKETING_COLLECTIONS === "true",
+    ENABLE_GRAVITY_MARKETING_COLLECTIONS === "true" ?? true,
```

Comment:
**suggestion**: How about `!== "false"` as the updated condition?

Reply:
hah, that works too! I overthought it